src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp

```diff
@@ -122,11 +122,12 @@ void medVtkViewItkDataImage4DInteractor::setInputData(medAbstractData *data)
 
         if (SetViewInput(data, layer) )
         {
+            double frameRate = static_cast<double>(m_poConv->getNumberOfVolumes()) / static_cast<double>(m_poConv->getTotalTime());
             d->imageData->setMetaData("SequenceDuration", QString::number(m_poConv->getTotalTime()));
-            d->imageData->setMetaData("SequenceFrameRate", QString::number((double)(m_poConv->getNumberOfVolumes() -1 )/ (double)m_poConv->getTotalTime()));
```

Comment:
What is the frame rate. If it is the number of volumes per unit of time, you should keep it as it was.

Reply:
framerate is now computed line 125 with this same computation except the "-1"